source/CILobbyMenu.cpp

```diff
@@ -137,8 +139,12 @@ bool LobbyMenu::init(const std::shared_ptr<cugl::AssetManager>& assets) {
     _gameStartBtn = std::dynamic_pointer_cast<scene2::Button>(assets->get<scene2::SceneNode>("lobby_startgamebutton"));
     _gameStartBtn->addListener([&](const std::string& name, bool down) {
         if (!down) {
+            _networkMessageManager->setGameStarted(1);
+            _networkMessageManager->sendMessages();
             _nextState = MenuState::LobbyToGame;
+            CULog("set listener state");
```

Comment:
NIT: I don't think we need this log. Or make it more specific if you want to keep it in